src/mappings/SetController.ts

```diff
@@ -0,0 +1,33 @@
+//https://github.com/subquery/tutorials-entity-relations/blob/main/src/mappings/AccountTransfers.ts
+import { SubstrateExtrinsic } from '@subql/types';
+import { Account } from '../types/models/Account';
+import { SetController } from '../types/models/SetController';
+
+async function ensureAccounts(accountIds: string[]): Promise<void> {
+    for (const accountId of accountIds) {
+        const account = await Account.get(accountId);
+        if (!account) {
+            await new Account(accountId).save();
+        }
+    }
+}
+
+export async function handleSetController(extrinsic: SubstrateExtrinsic): Promise<void> {
+    const {
+        extrinsic: {
+            signer: id,
+            args: args,
+        },
+    } = extrinsic;
+    let controller = args.toString();
```

Comment:
this is far from the approach we took so far, did you do it on purpose ? i.e. sub entities with name-value

Reply:
Yeah I figured out that the toString() call reliably just returned the value I'm interested in no matter the runtime version.